packages/nimbus/src/components/checkbox/checkbox.stories.tsx

```diff
@@ -140,13 +140,33 @@ export const InvisibleLabel: Story = {
   play: async ({ canvasElement, step, args }) => {
     const canvas = within(canvasElement);
     const htmlInput = canvas.getByTestId("test-checkbox");
-
     await step("Has alternative label", async () => {
       await expect(htmlInput).toHaveAttribute("aria-label", args["aria-label"]);
     });
   },
 };
 
+export const StyleProps: Story = {
+  args: {
+    children: "I have an inline margin of 40px",
+    // @ts-expect-error: data-testid is not a valid prop
```

Comment:
Shouldn't we support this prop so that TS doesn't complain?

Reply:
probably, but we should do it across the board, which is outside the scope of this pr